frontend/src/components/ContentPanel/ContentPanelPrompts/PromptIOBox/IOBoxBar/index.tsx

```diff
@@ -24,30 +24,33 @@ export const IOBoxBar: React.FC<IOBoxBarProps> = ({
     errors,
 }) => {
     return (
-        <div className="absolute top-10 flex flex-row z-1">
-            {deleteSelf ? (
+        <div className="absolute w-full px-5 flex flex-row justify-end items-center z-1">
+            {!locked && !errors ? (
                 <IOBoxButton
-                    onClick={() => deleteSelf?.()}
-                    name="Delete"
-                    colorPalette="red"
+                    icon="ArrowPathIcon"
+                    onClick={generate}
+                    name="Generate"
+                    colorPalette="primary"
                     visible={showButtons}
+                    errors={errors}
```

Comment:
This `errors` will always be falsy. Should the button here be disabled in case of errors, rather than not being rendered at all?

Reply:
Talking with the other devs, it was reasoned that similarly to the delete button, generate wouldn't need to be rendered if it's not usable.
Removed the redundant code pertaining to errors from IOBoxBar and IOBoxButton.